checks/runtimeCheck.go

```diff
@@ -0,0 +1,90 @@
+package checks
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+
+	"github.com/newrelic/newrelic-lambda-extension/util"
+)
+
+var (
```

Comment:
This looks like a separation of concerns situation. The code here that is reused by other checks should probably be in its own file and just have code relevant to the check in the `*_check.go` files. Maybe a `runtime.go` for all this runtime config stuff?

Reply:
Or `runtime_config.go`?

Reply:
Sounds good!